FFmpegInterop/StreamFactory.cpp

```diff
@@ -0,0 +1,366 @@
+//*****************************************************************************
+//
+//	Copyright 2015 Microsoft Corporation
```

Comment:
Some files have Copyright 2015 instead of 2019, is there a difference or is it just a mistake?

Reply:
I updated the date when copying to new files.